src/gui/AddressDialog.hpp

```diff
@@ -22,11 +22,12 @@
 #include <AddressBox.hpp>
 
 #include <QCheckBox>
+#include <QCoreApplication>
```

Comment:
If include order doesn't matter, please group them (if necessary, eg. QMPlay2 includes, Qt includes, C includes) and sort from the longest to the shortest (in new code) :D

In this case:
```
#include <QCoreApplication>
#include <QCheckBox>
#include <QDialog>
```